internal/policy/policy_test.go

```diff
@@ -409,3 +407,52 @@ func TestStateFindDelegationEntry(t *testing.T) {
 		assert.Equal(t, &tuf.Delegation{Name: "4", Paths: []string{"path1/subpath2/*"}, Terminating: false, Role: tuf.Role{KeyIDs: []string{"157507bbe151e378ce8126c1dcfe043cdd2db96e"}, Threshold: 1}}, entry)
 	})
 }
+
+func TestListRules(t *testing.T) {
+	repo, _ := createTestRepository(t, createTestStateWithPolicy)
```

Comment:
Can we test with delegated roles? Keep this one and move to two `t.Run`s.

Reply:
I have added another test for delegated roles, but this meant that I had to change createTestStateWithPolicy to use the root key, since using GPG keys for delegated entries makes them fail the state.Verify check.

Reply:
the leaf delegation can still point to a gpg key no?